Curs/src/events/commonEvents.js

```diff
@@ -1,6 +1,9 @@
 interviewApp.commonEvents = function() {
 
-    const goToTheLoginPage = function() {
+    const goToTheLoginPage = function(event) {
+        event.preventDefault();
```

Comment:
Can you explain what this does and why you need it?

Reply:
The event.preventDefault() method stops the default action of an element from happening.

Reply:
We need it because if we don't have it, the element (if we talk about a form for example) will try to execute the action, and if the form have no action the program will crush.

Reply:
:+1: